app/api/helpers/custom_fields.py

```diff
@@ -263,3 +263,20 @@ class Licence(Raw):
 
     def format(self, value):
         return DataGetter.get_licence_details(value)
+
+
+class PlaceHolder(CustomField):
+    """
+    Background Image PlaceHolder field for Events
+    """
+    __schema_example__ = 'http://url.com/img.ext'
+
+    def format(self, value):
+        custom_placeholder = DataGetter.get_custom_placeholder_by_name(value.sub_topic or value.topic or 'Other')
+        if custom_placeholder:
```

Comment:
It'd great if you can move this logic to a method
```
    def get_placeholder_by_event(event):
        custom_placeholder = DataGetter.get_custom_placeholder_by_name(event.sub_topic or event.topic or 'Other')
        if custom_placeholder:
            return custom_placeholder.url
        else:
            default_images = DataGetter.get_event_default_images()
            return url_for('static', filename='placeholders' + default_images[event.sub_topic or event.topic or 'Other'])
```
in data_getter.py so this can be reused whenever needed at other places